packages/plugin-ext/package.json

```diff
@@ -63,6 +62,7 @@
   "devDependencies": {
     "@theia/ext-scripts": "^0.8.0",
     "@types/decompress": "^4.2.2",
+    "@types/escape-html": "^0.0.20",
```

Comment:
Types should go with dependency. Please move it up to the `dependencies`.
By the way, other `@types/` should be moved too.

Reply:
Is there a reason for that? If so, it would be nice to write it down in the contribution guidelines.